crates/common/tedge_config/src/tedge_config_cli/tedge_config.rs

```diff
@@ -396,7 +396,13 @@ define_tedge_config! {
                 /// The port used for the local Cumulocity HTTP proxy
                 #[tedge_config(example = "8001", default(value = 8001u16))]
                 port: u16,
-            }
+            },
+
+            #[doku(as = "PathBuf")]
+            cert_path: Utf8PathBuf,
+
+            #[doku(as = "PathBuf")]
+            key_path: Utf8PathBuf,
```

Comment:
Doc is missing.

Reply:
Not due to this PR, but unfortunate. We are not consistent in our naming: there is a mix of `cert_path` and `cert_file` settings.